metadetect/metadetect.py

```diff
@@ -120,6 +146,23 @@ def _add_positions_and_psf(self, cat, res):
 
             newres['sx_col'] = cat['x']
             newres['sx_row'] = cat['y']
+
+            rows_noshear, cols_noshear = shearpos.unshear_positions(
+                newres['sx_row'],
+                newres['sx_col'],
+                shear_str,
+                obs,  # an example for jacobian and image shape
+            )
+
+            newres['sx_row_noshear'] = rows_noshear
+            newres['sx_col_noshear'] = cols_noshear
+
+            dims = obs.image.shape
+            rclip = rows_noshear.clip(min=0, max=dims[0]-1).astype('i4')
+            cclip = cols_noshear.clip(min=0, max=dims[1]-1).astype('i4')
```

Comment:
We probably want to do something like `(rows_noshear + 0.5).astype('i4').clip(min=0, max=dims[0]-1)` to get the nearest pixel.

Reply:
This function would work too: https://docs.scipy.org/doc/numpy/reference/generated/numpy.rint.html

Reply:
ok